low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yaml

```diff
@@ -0,0 +1,200 @@
+      - RealStorControllerSensor
+      - RealStorDiskSensor
+      - RealStorSideplaneExpanderSensor
+      - RealStorLogicalVolumeSensor
+      - IEMSensor
+      - NodeHWsensor
+      - RealStorEnclosureSensor
+      - SASPortSensor
+      - MemFaultSensor
+      - CPUFaultSensor
+
+   actuators:
+      - Service
+      - RAIDactuator
+      - Smartctl
+      - NodeHWactuator
+      - RealStorActuator
+
+   degraded_state_modules:
+      - ServiceWatchdog
+      - RAIDsensor
+      - NodeData
+      - IEMSensor
+      - NodeHWsensor
+      - DiskMsgHandler
+      - LoggingMsgHandler
+      - ServiceMsgHandler
+      - NodeDataMsgHandler
+      - NodeControllerMsgHandler
+      - RealStorActuatorMsgHandler
+      - SASPortSensor
+      - MemFaultSensor
+      - CPUFaultSensor
+
+RABBITMQINGRESSPROCESSOR:
+   virtual_host: SSPL
+   queue_name: actuator-req-queue
+   exchange_name: sspl-in
+   routing_key: actuator-req-key
+   username: sspluser
+   password: gAAAAABgBrJSxD2j1TI2HpQCY9yqvsp3Zxcaqt6mvD7Kz8uoOsx-PWrYOBBFHuSqYKvR1qIbL7ivOfEiUaly-8mwqtSslhwHGg==
+   primary_rabbitmq_host: localhost
+
+RABBITMQEGRESSPROCESSOR:
+   virtual_host: SSPL
+   queue_name: sensor-queue
+   exchange_name: sspl-out
+   routing_key: sensor-key
+   ack_queue_name: sensor-queue
+   ack_exchange_name: sspl-out
+   ack_routing_key: sensor-key
+   username: sspluser
+   password: gAAAAABgBrJSxD2j1TI2HpQCY9yqvsp3Zxcaqt6mvD7Kz8uoOsx-PWrYOBBFHuSqYKvR1qIbL7ivOfEiUaly-8mwqtSslhwHGg==
+   message_signature_username: sspl-ll
+   message_signature_token: ALOIUD986798df69a8koDISLKJ282983
+   message_signature_expires: 3600
+   iem_route_addr:
+   iem_route_exchange_name: sspl-out
+   primary_rabbitmq_host: localhost
+   limit_consul_memory: 50000000
+
+LOGGINGPROCESSOR:
+   virtual_host: SSPL
+   queue_name: iem-queue
+   exchange_name: sspl-in
+   routing_key: iem-key
+   username: sspluser
+   password: gAAAAABgBrJSxD2j1TI2HpQCY9yqvsp3Zxcaqt6mvD7Kz8uoOsx-PWrYOBBFHuSqYKvR1qIbL7ivOfEiUaly-8mwqtSslhwHGg==
+   primary_rabbitmq_host: localhost
+
+RABBITMQCLUSTER:
+   cluster_nodes: localhost
+   erlang_cookie: QLDZYPYEYGHECTHYQXFJ
+
+LOGGINGMSGHANDLER:
+   iem_routing_enabled: false
+   iem_log_locally: true
+
+DISKMSGHANDLER:
+   dmreport_file: /tmp/sspl/drivemanager/drive_manager.json
+   always_log_iem: false
+   max_drivemanager_events: 14
+   max_drivemanager_event_interval: 10
+
+NODEDATAMSGHANDLER:
+   transmit_interval: 10
+   units: MB
+   disk_usage_threshold: 80
+   cpu_usage_threshold: 80
+   host_memory_usage_threshold: 80
+
+RAIDSENSOR:
+   monitor: true
+   threaded: true
+   RAID_status_file: /proc/mdstat
+
+IPMI:
+   user: admin
+   pass: admin
+
+BMC_INTERFACE:
+   default: system
+
+REALSTORSENSORS:
+   monitor: true
+   polling_frequency: 30
+
+REALSTORPSUSENSOR:
+   threaded: true
+
+REALSTORFANSENSOR:
+   threaded: true
+
+REALSTORCONTROLLERSENSOR:
+   threaded: true
+
+REALSTORDISKSENSOR:
+   threaded: true
+   polling_frequency_override: 0
```

Comment:
lets add this 'polling_frequency_override' for all realstor sensors, even if its not in existing sspl.conf currently. Code already checks for it, and applies it if available. In LR R2, clear configuration for polling frequency is needed per sensor.

Reply:
Done